tests/components/garmin_connect/test_config_flow.py

```diff
@@ -91,8 +89,9 @@ def mock_garmin():
 
 async def test_abort_if_already_setup(hass, mock_garmin_connect):
     """Test abort if already setup."""
-    entry = MockConfigEntry(domain=DOMAIN, data=MOCK_CONF)
-    entry.add_to_hass(hass)
+    result = await hass.config_entries.flow.async_init(
```

Comment:
Pass the unique_id as a keyword argument to the mock config entry instead.

https://github.com/home-assistant/home-assistant/blob/7d9c8fdfa09323eef42a73dca295ab5fabe1d833/tests/common.py#L656-L672

Reply:
Done, thanks again for being super helpful!